lib/one_gadget/emulators/aarch64.rb

```diff
@@ -44,6 +44,12 @@ def argument(idx)
         registers["x#{idx}"]
       end
 
+      def get_corresponding_stack(obj)
```

Comment:
Add documents like other functions, include the description that `obj` must be the register "sp"